app/views/idv/review/new.html.slim

```diff
@@ -32,7 +32,9 @@ p.italic = t('idv.messages.review.intro')
   .h6 Phone number
   .h4.bold #{idv_params[:phone]}
 
-= form_tag('/idv/review', method: 'put')
+= simple_form_for(current_user, url: idv_review_path,
+    html: { autocomplete: 'off', method: :put, role: 'form' }) do |f|
+  = f.input :password, label: t('idv.form.password'), required: 'true'
```

Comment:
@jessieay is right. We use `true` everywhere else. I believe the bug you are thinking about is the one I found where we weren't using Simple Form, and were using `require` instead of `required`.